lib/compat/wordpress-6.7/rest-api.php

```diff
@@ -115,6 +115,45 @@ function gutenberg_override_default_rest_server() {
 }
 add_filter( 'wp_rest_server_class', 'gutenberg_override_default_rest_server', 1 );
 
+/**
+ * Updates the comment type in the REST API for WordPress version 6.7.
```

Comment:
Should all `6.7` references be changed to `6.8` now?

Reply:
Done